### ISSUE TITLE
pix2pix example for flutter

### ISSUE THREAD

**Issue by Guidosalimbeni** [shaqian/flutter_tflite#137]
Hi,
just wondering if you can point me to an example of a keras or tf pix2pix implementation that can easily converted into a tflite model to be used in flutter? I have been having issues to overcome the batchnormalisation layer not recognised and I presume that you tested with some example to test the compatibility . would be great if you can share the code.. thanks
Guido
